### PR TITLE
Upgrade nebula sdk

### PR DESCRIPTION
**Pull request by MegaByte875**

### CODE REVIEW

File: pkg/nebula/meta_client.go [hunk 1]
```diff
@@ -22,25 +22,25 @@ import (
 
 	"github.com/pkg/errors"
 
-	"github.com/vesoft-inc/nebula-go/nebula"
-	"github.com/vesoft-inc/nebula-go/nebula/meta"
+	"github.com/vesoft-inc/nebula-go/v2/nebula"
+	"github.com/vesoft-inc/nebula-go/v2/nebula/meta"
+
```
Comment (veezhang) [inline pkg/nebula/meta_client.go:27]: import 4 groups, remove this line?

Comment (MegaByte875) [reply pkg/nebula/meta_client.go:26]: ok

File: config/samples/apps_v1alpha1_nebulacluster.yaml [hunk 2]
```diff
@@ -13,11 +13,11 @@ spec:
         memory: "1Gi"
     replicas: 1
     image: vesoft/nebula-graphd
-    version: v2.0.1
+    version: v2.5.1
```
[the remark this answers is not in the document]
Comment (MegaByte875) [reply config/samples/apps_v1alpha1_nebulacluster.yaml]: ok

File: pkg/controller/component/pvc_gc.go [hunk 3]
```diff
@@ -57,26 +57,33 @@ func PvcMark(pvcClient kube.PersistentVolumeClaim, component v1alpha1.NebulaClus
...
-		pvcName := ordinalPVCName(component.Type(), componentName, i)
-		pvc, err := pvcClient.GetPVC(component.GetNamespace(), pvcName)
-		if err != nil {
-			return fmt.Errorf("get pvc %s for cluster %s/%s failed: %s", pvcName, component.GetNamespace(), component.GetClusterName(), err)
-		}
+		pvcNames := ordinalPVCNames(component.Type(), componentName, i)
+		for _, pvcName := range pvcNames {
+			pvc, err := pvcClient.GetPVC(component.GetNamespace(), pvcName)
+			if err != nil {
+				return fmt.Errorf("get pvc %s for cluster %s/%s failed: %s", pvcName, component.GetNamespace(), component.GetClusterName(), err)
```
[the remark this answers is not in the document]
Comment (MegaByte875) [reply pkg/controller/component/pvc_gc.go:64]: ok

File: pkg/controller/component/pvc_gc.go [hunk 4]
```diff
@@ -57,26 +57,33 @@ func PvcMark(pvcClient kube.PersistentVolumeClaim, component v1alpha1.NebulaClus
...
 	}
 	return nil
 }
 
-func ordinalPVCName(componentType v1alpha1.ComponentType, setName string, ordinal int32) string {
-	return fmt.Sprintf("%s-%s-%d", componentType, setName, ordinal)
+func ordinalPVCNames(componentType v1alpha1.ComponentType, setName string, ordinal int32) []string {
+	logPVC := fmt.Sprintf("%s-log-%s-%d", componentType, setName, ordinal)
+	dataPVC := fmt.Sprintf("%s-data-%s-%d", componentType, setName, ordinal)
+	if componentType == v1alpha1.GraphdComponentType {
```
[the remark this answers is not in the document]
Comment (MegaByte875) [reply pkg/controller/component/pvc_gc.go:85]: I got it

File: pkg/controller/component/storaged_scaler.go [hunk 5]
```diff
@@ -128,18 +128,21 @@ func (ss *storageScaler) ScaleIn(nc *v1alpha1.NebulaCluster, oldReplicas, newRep
...
-	var deleted bool
-	pvcName := ordinalPVCName(nc.StoragedComponent().Type(), nc.StoragedComponent().GetName(), newReplicas)
-	_, err = ss.clientSet.PVC().GetPVC(nc.GetNamespace(), pvcName)
-	if apierrors.IsNotFound(err) {
-		deleted = true
+	var deleted = true
+	pvcNames := ordinalPVCNames(nc.StoragedComponent().Type(), nc.StoragedComponent().GetName(), newReplicas)
+	for _, pvcName := range pvcNames {
+		_, err = ss.clientSet.PVC().GetPVC(nc.GetNamespace(), pvcName)
+		if !apierrors.IsNotFound(err) {
```
[the remark this answers is not in the document]
Comment (MegaByte875) [reply pkg/controller/component/storaged_scaler.go]: I got it

**Review by veezhang — APPROVED** [review]
LGTM

**Pull request merged**